modules/sandboxed-containers-collecting-data.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * sandboxed_containers/troubleshooting-sandboxed-containers.adoc
+
+//This file contains UI elements and/or package names that need to be updated.
+
+[id="sandboxed-containers-collecting-data_{context}"]
+= About collecting {sandboxed-containers-first} data
+
+You can use the `oc adm must-gather` CLI command to collect information about your
+cluster, including features and objects associated with {sandboxed-containers-first}:
+
+* All namespaces (and their child objects) that belong to any {sandboxed-containers-first}
```

Comment:
Per the IBM style guide: Use parentheses to identify items such as abbreviations, symbols, and measurements, but avoid using parentheses in general text.

Reply:
Okay, I'll change it. I had copied this content from another location in the OCP help, so I had relied on that being correct...